gsy-node/modules/gsy-collateral/src/mock.rs

```diff
@@ -34,16 +32,13 @@ impl system::Config for Test {
 	type BlockWeights = ();
```

Comment:
You have also this macro #[derive_impl(frame_system::config_preludes::TestDefaultConfig)] for frame_system